app/controllers/api/v1/road_trip_controller.rb

```diff
@@ -1,23 +1,23 @@
 class Api::V1::RoadTripController < ApplicationController 
 
   def create
-    user = User.find_by(api_key: params[:body][:api_key])
+    user = User.find_by(api_key: params[:api_key])
```

Comment:
no need to send it as 'body'. I didn't understand the original request. Can be passed in the 'body' of postman